bin/distmap

```diff
@@ -288,12 +289,13 @@ if ( $hadoop_home eq "" ) {
 }
 
 
-if ( $readtools =~ /\.jar$/ ) {
-  $readtools = "eval java \\\$JAVA_OPTS -jar $readtools";
-}
-
-unless ( $readtools ne "" and `$readtools --version` gt "1.2.1" ) {
-	pod2usage(-msg=>"\n\tERROR: only found ReadTools \"$readtools\", which does not provide version >= 1.2.1");
+if ( $readtools_arg eq "" ) {
+	pod2usage(-msg=>"\n\tERROR: no ReadTools found. Use --readtools or add a wrapper/jar to your PATH");
+} else {
+	$readtools = ReadToolsUtils::get_readtools_runnable_cmd($readtools_arg);
+	if ( `$readtools --version` gt ReadToolsUtils::VERSION ) {
```

Comment:
Really "gt" ?

Reply:
That was the previous behavior (see the diff in red)

Reply:
Sorry, I just realized that it's true... it should be `lt` because before it was `unless`. Will change tomorrow

Reply:
Done.